src/test/scala/com.gu.contentapi.client/parser/JsonParserItemTest.scala

```diff
@@ -210,6 +210,17 @@ class JsonParserItemTest extends FlatSpec with Matchers with ClientTest {
 
   it should "parse the publication dates of blocks" in {
     contentItemWithBlocksResponse.content.get.blocks.get.main.get.firstPublishedDate should be(Some(new DateTime("2015-04-09T14:27:28.486+01:00")))
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.createdDate should be(Some(new DateTime("2015-04-09T14:27:28.486+01:00")))
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.lastModifiedDate should be(Some(new DateTime("2015-04-09T14:27:35.492+01:00")))
+  }
+
+  it should "parse the users of blocks" in {
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.createdBy.get.email should be("mariot.chauvin@theguardian.com")
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.createdBy.get.firstName should be (None)
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.createdBy.get.lastName should be (None)
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.lastModifiedBy.get.email should be("david.blishen@theguardian.com")
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.lastModifiedBy.get.firstName should be(Some("David"))
+    contentItemWithBlocksResponse.content.get.blocks.get.main.get.lastModifiedBy.get.lastName should be(Some("Blishen"))
   }
```

Comment:
This is not necessarily to do with your pull request, but is it worth tidying this test up a little? For example: 

  it should "parse the users of blocks" in {

    val mainBlock = contentItemWithBlocksResponse.content.get.blocks.get.main.get

    mainBlock.createdBy.get.email should be("mariot.chauvin@theguardian.com")
    mainBlock.createdBy.get.firstName should be (None)
    mainBlock.get.createdBy.get.lastName should be (None)
    mainBlock.lastModifiedBy.get.email should be("david.blishen@theguardian.com")
    mainBlock.lastModifiedBy.get.firstName should be(Some("David"))
    mainBlock.lastModifiedBy.get.lastName should be(Some("Blishen"))
   }

You could even pull some of the more common ones into helper methods for extracting these types of things to avoid all the duplication in the file. I can do this when I add editions here if you want.